apis/bases/client.openstack.org_openstackclients.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.10.0
```

Comment:
@jlarriba I think these are the problem for precommit.  We need to run `make manifests` with the newer version of `controller-gen` and push that to the PR.